README.md

```diff
@@ -14,8 +14,8 @@ engines:
 
 ## Configuration
 
-This engine accepts `tags` and `cookbook_paths` in its configuration. Both
-values are optional:
+This engine accepts `tags`, `include_rules` and `cookbook_paths` in its
```

Comment:
small nit: can we keep ordering of these attrs consistent between the list here & the example config below?